src/main/java/com/google/sps/servlets/HistoryServlet.java

```diff
@@ -19,6 +19,9 @@
 import com.google.sps.data.TutorSession;
 import com.google.sps.data.SampleData;
 import com.google.sps.data.Student;
+import com.google.sps.utilities.StudentsDatastoreService;
```

Comment:
I think you saw in my PR, I just used the TutorSession datastore service for history and rating because they basically used the same methods.

Reply:
Yeah, I thought that was a better approach